contracts/base/LoansBase.sol

```diff
@@ -345,7 +324,28 @@ contract LoansBase is Base {
         uint256 tokensIn
     ) internal;
 
-    function _getCollateralInfo(uint256 totalOwed, uint256 collateralRatio)
+    function _getCollateralInfo(uint256 loanID)
```

Comment:
Does this function make calls less efficient? The places that call `_getCollateralInfo` seem to only use 1/2 of the outputs it gives and so don't need the other values to be calculated.

Maybe this is an improvement for later

Reply:
The output variables that are being used require the other output variables to be calculated internally. The difference is now we are returning all the variables.
The function was created using the previous logic. It just put together two functions (_getCollateralNeededInTokens and _convertTokenToWei)
For sure, we can improve it later.